src/formatError.ts

```diff
@@ -0,0 +1,11 @@
+import {Steps} from './assertOrder'
+
+export class FormatError {
```

Comment:
It is not likely that we need states during formatting, so exporting the function directly would be a better option.